ang/civicase/CaseCard--long.html

```diff
@@ -0,0 +1,33 @@
+<div
+  class="panel panel-default panel-strip-primary civicase__case-card civicase__case-card--long"
+  ng-style="{'border-left-color': data.color}"
+  ng-class="{'civicase__case-card--closed': data.status === 'Resolved'}
+  ">
+  <div class="panel-body">
+    <div class="civicase__case-card__row civicase__case-card__row--primary clearfix">
```

Comment:
I think its better to consider `civicase__case-card__row` as the primary. And only have one variant i.e. `civicase__case-card__row--scondary`.